fellow-submissions/mark-griffith/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <link rel="stylesheet" href="https://unpkg.com/blocks.css/dist/blocks.min.css" />
+    <title>The True Test</title>
+</head>
+<body>
+    <main>
+        <section>
+            <header>
+                <h1>The True Test For Greatness</h1>
+                <p>Answer this correctly for a prize!</p>
+            </header>
+            <fieldset>
+                <legend>Games</legend>
+                <label for="#">What game is the question below referencing:
+                    <input type="radio" name="game" value="Skyrim"> Skyrim
+                    <input type="radio" name="game" value="Fortnite"> Fortnite
+                    <input type="radio" name="game" value="Minecraft"> Minecraft
+                </label>
+                <label for="#">What is the music of life:
+                    <input type="text" name="answer"/>
+                    <input type="submit" value="Submit"/>
+                </label>
+            </fieldset>
+        </section>
+
+        <section>
+            <fieldset>
+                <legend>Anime</legend>
+                <label for="">What is the best anime ever:
```

Comment:
All you have to do is set a value for your `for` attribute. Then you will need to match it to the `id` of your `<select>` tag. Otherwise, this is good.